### PR TITLE
adding unlock-abi-1-0

### PR DESCRIPTION
**Pull request by julien51**
# Description

All in the title!

# Checklist:

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [X] This PR only contains configuration changes (package.json, etc.)
  - [ ] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [ ] My code follows the style guidelines of this project, including naming conventions
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: unlock-js/package.json [hunk 1]
```diff
@@ -17,7 +17,8 @@
   "author": "",
   "license": "MIT",
   "dependencies": {
-    "ethers": "4.0.27"
+    "ethers": "4.0.27",
+    "unlock-abi-1-0": "^1.0.0"
```
Comment (cellog) [inline unlock-js/package.json]: devDependencies?

Comment (julien51) [reply unlock-js/package.json]: Oh! That is right!

**Comment by julien51** [comment]
Thanks @cellog PTAL!

File: unlock-js/package.json [hunk 2]
```diff
@@ -29,14 +29,15 @@
     }
   },
   "devDependencies": {
-    "jest": "^24.1.0",
-    "nock": "^10.0.5",
     "@babel/cli": "^7.2.3",
     "@babel/core": "^7.3.4",
     "@babel/preset-env": "^7.4.2",
     "babel-jest": "^24.7.1",
+    "jest": "^24.1.0",
```
Comment (julien51) [inline unlock-js/package.json]: Not sure why npm re-ordered things :/

Comment (cellog) [reply unlock-js/package.json]: they were probably done by hand before (by me...) and now they are alphabetical

**Pull request merged**